commcare_connect/opportunity/forms.py

```diff
@@ -236,9 +236,9 @@ def save(self, commit=True):
         self.instance.created_by = self.user.email
         self.instance.modified_by = self.user.email
 
-        # In the case of a managed opportunity, the organization will be null initially,
-        # as it will be set to the network manager's organization when the invitation is accepted.
-        if not self.managed_opp:
+        if self.managed_opp:
+            self.instance.organization = self.cleaned_data.get("organization")
```

Comment:
since we always set this, we should no longer make it nullable.

Reply:
I made the changes here since program 0001 has been modified multiple times in previous PRs, and both nodes are dependent on each other. To avoid conflicts across branches, I implemented the changes in this branch. As here the script was last modified. [e43b07f](https://github.com/dimagi/commcare-connect/pull/385/commits/e43b07fe23a5f152f05d0d62fdd7d98cfc21499b)